src/main/java/de/hs_mannheim/informatik/ct/util/DocxTemplate.java

```diff
@@ -96,7 +100,7 @@ public XWPFDocument generate(List<T> dataSource) throws IOException, XmlExceptio
                     for (int paragraphIndex = 0; paragraphIndex < paragraphs.value.size(); paragraphIndex++) {
                         val paragraph = document.createParagraph();
                         paragraph.getCTP().set(paragraphs.value.get(paragraphIndex));
-                        val isLastPage = paragraphs.index == dataSource.size() - 1;
+                        val isLastPage = paragraphs.index == 0;
                         if (!isLastPage && paragraphIndex == paragraphs.value.size() - 1) {
```

Comment:
Since there is only one page, it is always the last page, so no page breaks necessary.